apps/ashley/vars/all/main.yml

```diff
@@ -37,6 +37,12 @@ ashley_django_settings_module: "settings"
 ashley_django_configuration: "Development"
 ashley_secret_name: "ashley-{{ ashley_vault_checksum | default('undefined_ashley_vault_checksum') }}"
 
+# -- haystack / elasticsearch settings
+ashley_elasticsearch_host: "elasticsearch"
+ashley_elasticsearch_port: "9200"
+ashley_elasticsearch_index: "ashley"
+ashley_cron_reindex_schedule: "17 4,12,20 * * *"
```

Comment:
Can you explain or add a comment to justify this frequency?

Reply:
Oh, I had our use case in mind when writing this.
Maybe I should set `0 * * * *` as a default value to launch it every hour + a comment. It's easier to understand.
WDYT @jmaupetit  ?

Reply:
Yes !